cupy_backends/cuda/stream.pyx

```diff
@@ -50,11 +52,12 @@ cdef intptr_t get_current_stream_ptr():
     return <intptr_t>tls.get_current_stream_ptr()
 
 
-cdef set_current_stream_ptr(intptr_t ptr):
+cdef set_current_stream_ptr(intptr_t ptr, int dev=-1):
```

Comment:
Could you use `device_id` instead of `dev` for consistency with `cupy.cuda.Memory*` APIs?

Reply:
Great! I forgot that, thanks.

Reply:
Done in 253510f.